py3status/modules/xsel.py

```diff
@@ -32,13 +38,25 @@ class Py3status:
     """
     # available configuration parameters
     cache_timeout = 0.5
-    command = 'xsel -o'
+    command = 'xsel -output'
+    click_command = 'xsel --clear --clipboard'
```

Comment:
@tobes (Nitpicking) We could support `on_click any = 'xsel --clear --clipboard'`

Reply:
We could but I'm not a fan.  Maybe `on_click 1 2 3 4 5 = 'xsel --clear --clipboard'` could be done at some point as more flexible.  I don't see this as a priority.

I do agree that this should be done via the config rather than as part of the module.

Reply:
Ohh nice, I was not aware of that functionality.